### DISCUSSION TITLE
Gamechanger: Dreambooth fork that can train all new models and merges

### DISCUSSION THREAD

**Discussion by Omenizer** [TheLastBen/fast-stable-diffusion#1671] [Ideas]
There's a fork by theovercomer8 that can train all of the new models and custom merges.

Up to now, only SD base models and a handful early finetunes worked, and we had to add difference to newer models. 
But with this fork, almost all models (all I tried) can be converted to diffusers and trained!

Any chance this could make it into @TheLastBen Fast DB? 

https://github.com/theovercomer8/sd-playground

**Comment by TheLastBen** [comment]
does it convert v2-768 models correctly ?

**Reply by Omenizer** [reply]
Don't know, I only use 1.5. 

But the author has his own v2 model (Illuminati diffusion, the one with noise offset) so I'd think it does. All presets are v2 for default too.

**Reply by TheLastBen** [reply]
The conversion script he uses is the default huggingface diffusers script, I tested it and it doesn't work with the v2-768
